src/xenia/kernel/util/xlast.cc

```diff
@@ -133,6 +138,87 @@ XLastMatchmakingQuery* XLast::GetMatchmakingQuery(const uint32_t query_id) {
   return new XLastMatchmakingQuery(node);
 }
 
+uint32_t XLast::GetPresenceStringId(const uint32_t context_id) {
```

Comment:
Global idea for Xlast and so on. Instead of returning 0 as default we can use std::optional<uint32>